### PR TITLE
[EDU-3855] work with node services variables

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue:
[EDU-3855](https://aziontech.atlassian.net/browse/EDU-3855)

[EDU-3855]: https://aziontech.atlassian.net/browse/EDU-3855?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 1]
```diff
@@ -0,0 +1,215 @@
+---
+title: Work with variables
+description: aaa
+meta_tags: deploy, orchestrator
+namespace: docs_deploy_work_with_variables
+menu_namespace: deployMenu
+permalink: /documentation/products/guides/deploy/work-with-variables/
+---
+
+During the deployment process on Edge Orchestrator, we have the possibility to work with variables. A variable can be set on an [edge service](#edge-services-variables) and on an [edge node](#edge-node).
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:10]:
```suggestion
During the deployment process on **Edge Orchestrator**, you can work with variables. A variable can be set on an [edge service](#edge-services-variables) and an [edge node](#edge-node).
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:10]: A variable can be set on an [edge service](#edge-services-variables) or an [edge node](#edge-node).
OU
 A variable can be set on an [edge service](#edge-services-variables) and an [edge node](#edge-node).

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+:::note
+The priority is always the edge node variable. So, if you have a variable configured on an edge service and the same variable configured on an edge node, the one on the node will override the one configured on the edge service.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:13]:
```suggestion
The edge node variable always has priority. So, if you have a variable configured on an edge service and the same variable configured on an edge node, the one on the node will override the one configured on the edge service.
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+:::
+
+You can create edge node and edge services variables using:
+
+- [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/)
+- [Azion API](https://api.azion.com/)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:19]:
```suggestion
- [Real-Time Manager (RTM)](#via-rtm)
- [Azion API](#via-api)
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+
+---
+
+## Edge Services variables
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:23]:
```suggestion
## Creating Edge Services variables 
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx (hunk 1 above)
Comment (gabriel-azion) [reply src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:10]: and @MarianaAguilera

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+---
+
+## Edge Services variables 
+
+### Via RTM 
+
+1. Go to [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Libraries** > **Edge Services**.
+2. Select the edge service to which you want to add a variable.
+3. Go to the **Environment** tab.
+5. Add the following variable:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:30]:
```suggestion
4. Add the following variable:
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+--data '{
+  "content_type": "Text",
+  "name": "/resource",
+  "content": "I am using {{name}} edge orchestrator"
+}'
+```
+
+---
+
+## Edge Node
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:107]:
```suggestion
## Edge Node variables
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+1. Go to [Real-Time Manager (RTM)](/en/documentation/products/guides/how-to-access-rtm/) > **Edge Libraries** > **Edge Services**.
+2. Select the edge service to which you want to add a variable.
+3. Go to the **Environment** tab.
+5. Add the following variable:
+
+```bash 
+name=azion
+```
+
+6. Click the **Save** button.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:36]:
```suggestion
5. Click the **Save** button.
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+  "variables": [
+    {
+      "name": "name",
+      "value": "azion"
+    }
+  ]
+}'
+```
+
+Now, you've created a variable, related to a specific service. You need to access this variable through a resource.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:80]:
```suggestion
Now, you've created a variable related to a specific service. You need to access this variable through a resource.
```

File: src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+4. Choose the service to which you want to apply the variable.
+5. Add the following content to the variables block: 
+
+```bash 
+name=azion
+```
+
+6. Click the **Save** button.
+
+Now, you've created a variable, related to a specific node. You need to access this variable through a resource.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/deploy-journey/work-with-variables/work-with-variables.mdx:123]:
```suggestion
Now, you've created a variable related to a specific node. You need to access this variable through a resource.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**